modules/docs/src/main/mdoc/docs/sleep.md

```diff
@@ -17,45 +17,65 @@ trait Sleep[M[_]] {
 }
 ```
 
-Out of the box, the core module provides instances for `Id` and `Future` that
-simply do a `Thread.sleep(...)`.
+Out of the box, the core module provides instances for any type with an implicit cats-effect
+[`Timer`](https://typelevel.org/cats-effect/datatypes/timer.html) in scope.
 
-Note: these instances are not provided if you are using Scala.js, as
-`Thread.sleep` doesn't make any sense in JavaScript.
+For example using `cats.effect.IO`:
 
 ```scala mdoc:silent:reset-class
 import retry.Sleep
-import cats.Id
+import cats.effect.{IO, Timer}
 import scala.concurrent.duration._
+import scala.concurrent.ExecutionContext.global
 
-Sleep[Id].sleep(10.milliseconds)
+implicit val timer: Timer[IO] = IO.timer(global)
+
+Sleep[IO].sleep(10.milliseconds)
 ```
 
+Or if you're using an abstract `F[_]`:
+
 ```scala mdoc:silent:reset-class
 import retry.Sleep
-import scala.concurrent.Future
+import cats.effect.Timer
 import scala.concurrent.duration._
-import scala.concurrent.ExecutionContext.Implicits.global
 
-Sleep[Future].sleep(10.milliseconds)
+def sleepWell[F[_]: Timer] =
+  Sleep[F].sleep(10.milliseconds)
 ```
 
-The `cats-effect` module provides an instance that uses a cats-effect
-[`Timer`](https://typelevel.org/cats-effect/datatypes/timer.html).
+
+Being able to inject your own `Sleep` instance can be handy in tests, as you
+can mock it out to avoid slowing down your unit tests.
+
+
+## alleycats-retry
+
+The `alleycats-retry` module provides instances for `cats.Id`, `cats.Eval` and `Future` that
+simply do a `Thread.sleep(...)`.
+
+To use them, simply import `retry.alleycats.instances`:
```

Comment:
The import is missing a `._` on the end.

Also let's include a line here showing how to add a dependency on the alleycats module. It's a bit of a mouthful:

```
```scala mdoc:passthrough
println(
  s"""
  |```
  |libraryDependencies += "com.github.cb372" %% "alleycats-retry" % "${retry.BuildInfo.version.replaceFirst("\\+.*", "")}"
  |```
  |""".stripMargin.trim
)
```
```